std/io/streams.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+export function toWriter(
+  streamWriter: WritableStreamDefaultWriter<Uint8Array>
+): Deno.Writer {
+  return {
+    async write(p: Uint8Array): Promise<number> {
+      await streamWriter.ready;
+      await streamWriter.write(p);
+      return p.length;
+    },
+  };
+}
+
+export function toReader(
```

Comment:
Having it in a file called `streams.ts` changes the naming situation a bit, but I'd still go with `fromStreamReader()`/`fromStreamWriter()`. The intuitive type is `stream-reader -> reader`, `toReader()` doesn't communicate that well.

Reply:
Sounds good to me.